ignite/services/network/networkchain/request.go

```diff
@@ -0,0 +1,60 @@
+package networkchain
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	cosmosgenesis "github.com/ignite/cli/ignite/pkg/cosmosutil/genesis"
+	"github.com/ignite/cli/ignite/pkg/events"
+	"github.com/ignite/cli/ignite/services/network/networktypes"
+)
+
+// CheckRequestChangeParam builds the genesis for the chain from the launch approved requests
+func (c Chain) CheckRequestChangeParam(
+	ctx context.Context,
+	module, param string, value []byte,
+) error {
+	c.ev.Send("Checking the param change", events.ProgressStart())
+
+	if err := c.initGenesis(ctx); err != nil {
+		return err
+	}
+
+	genesisPath, err := c.chain.GenesisPath()
+	if err != nil {
+		return errors.Wrap(err, "genesis of the blockchain can't be read")
```

Comment:
can we use only the `fmt.Errof` or only the `errors.Wrap`?

https://github.com/ignite/cli/pull/3208/files#r1037391887